lib/global-admin/addon/accounts/new/controller.js

```diff
@@ -66,6 +72,9 @@ export default Controller.extend(NewOrEdit, {
   validate() {
     var errors = [];
 
+    if (this.canUserLogIn && !this.canUserLogIn()) {
+      errors.pushObject(this.intl.findTranslationByKey('formGlobalRoles.loginError'));
```

Comment:
The surrounding code is making use of push. Any reason to diverge from what's already here? If you want to just use pushObject everywhere maybe go ahead and change the surrounding code to use pushObject.